azure-pipelines.yml

```diff
@@ -151,8 +149,11 @@ stages:
       - ${{ if or(eq(variables['System.TeamProject'], 'public'), in(variables['Build.Reason'], 'PullRequest')) }}:
         - job: Linux
           pool:
-            name: NetCore1ESPool-Svc-Public
-            demands: ImageOverride -equals Build.Ubuntu.1804.Amd64.Open
+            ${{ if eq(variables['System.TeamProject'], 'public') }}:
+              vmImage: ubuntu-18.04
+            ${{ if eq(variables['System.TeamProject'], 'internal') }}:
+              name: NetCore1ESPool-Svc-Internal
```

Comment:
6.0.2xx is not in servicing yet, we should use `NetCore1ESPool-Internal` probably